### ISSUE TITLE
Capitalization non-uniformity in display schema

### ISSUE THREAD

**Issue by russell-taylor** [OSVR/OSVR-JSON-Schemas#11]
The "Version" and "Note" fields in hmd/device/properties start with a capitol letter.  The other elements in there don't.  Did not check to see if this happens elsewhere.

This is starting to get baked into parsers...

**Comment by rpavlik** [comment]
Yeah, that's the least of the problems with that schema. Already tried fixing it once before realizing it broke every single app. It's already been baked into parsers in Unity and elsewhere - hence why we're pulling that parsing inside so we can change it up later.
